test/integration/content-picker/ContentPicker.e2e.test.js

```diff
@@ -11,51 +13,89 @@ const helpers = {
         cy.getByTestId('be-sub-header').contains('Codepen');
     },
     getRow: rowNum => cy.getByTestId('content-picker').find(`.bcp-item-row-${rowNum}`),
-    selectRow: row =>
-        row
+    selectRow(rowNum) {
+        this.getRow(rowNum)
+            .as('row')
             .click()
             .find('input[type="checkbox"]')
-            .should('be.checked'),
-    unselectRow: row =>
-        row
+            .should('be.checked');
+
+        cy.get('@row').find('.bcp-shared-access-select');
+
+        return cy.get('@row');
+    },
+    unselectRow(rowNum) {
+        this.getRow(rowNum)
+            .as('row')
             .click()
             .find('input[type="checkbox"]')
-            .should('not.be.checked'),
+            .should('not.be.checked');
+
+        cy.get('@row')
+            .find('.bcp-shared-access-select')
+            .should('not.exist');
+    },
 };
 
 describe('ContentPicker', () => {
+    beforeEach(() => {
+        cy.server();
+
+        cy.route(
+            'GET',
+            '**/folders/51964781421?fields=id,name,type,size,parent,extension,permissions,path_collection,modified_at,created_at,modified_by,has_collaborations,is_externally_owned,item_collection&direction=asc&limit=50&offset=0&sort=name',
+            'fixture:content-picker/root-folder.json',
+        );
+
+        cy.route(
+            'GET',
+            '**/folders/51965506671?fields=id,name,type,size,parent,extension,permissions,path_collection,modified_at,created_at,modified_by,has_collaborations,is_externally_owned,item_collection&direction=asc&limit=50&offset=0&sort=name',
+            'fixture:content-picker/sample-folder.json',
+        );
+
+        ['319004423111', '308566419514', '308409990441'].forEach(fileId => {
+            cy.route('GET', `**/files/${fileId}?fields=allowed_shared_link_access_levels,shared_link`, {
+                ...getSharedLinkRsp,
+                id: fileId,
+            });
+
+            cy.route('PUT', `**/files/${fileId}`, {
+                ...createSharedLinkRsp,
```

Comment:
I think we want to avoid explicit imports when possible. Can we import these via `cy.fixture`, instead?